DigitalLearningSolutions.Web/Views/Support/Support.cshtml

```diff
@@ -32,26 +34,26 @@
     <p class="nhsuk-lede-text nhsuk-u-margin-top-4">Access Digital Learning Solutions support in 5 easy steps...</p>
     <ol>
       <li>
-        Check the online Help documentation. It is searchable and comprehensive.
+        <a href="@configuration["CurrentSystemBaseUrl"]/help/Introduction.html" target="_blank">Check the online Help documentation</a>. It is searchable and comprehensive.

```

Comment:
Given that these links are now being set in two places, can we pull them into the view model?